Readme.md

```diff
@@ -45,4 +45,66 @@ $downloader->fetch($criteria, function(Email $email) {
 You can return false in callback function. In this case this email will be fetched also in next time. For processing emails you will need to create folder **INBOX/processed**.
 There is possiblity to setup criteria for fetching emails with *MailCriteria*. More information in source code.
 
+Some extended features:
+
+``` php
+
+// define what you consider your inbox folder..
+$downloader->setInboxFolder('INDBOX');
+
+// if processed folder does not exist, automatically try to create it
+$downloader->setProcessedFoldersAutomake(true);
+
+
+//// define what data to get for email
+//  FETCH_OVERVIEW      just basic header information (see imap_fetchoverview)
+//  FETCH_BODY          email body as string
+//  FETCH_HEADERS       complete email headers as string
+// default = FETCH_OVERVIEW | FETCH_BODY
+
+$fetchParts = Downloader::FETCH_OVERVIEW | Downloader::FETCH_BODY | Downloader::FETCH_HEADERS;
```

Comment:
This variable is never used in this example. I understand that you will be able to set behaviour for downloader based on combination of this settings. But I think it will be valuable to use it in example, not just define variable.